students/Mintae_K/lesson04/mailroom2.py

```diff
@@ -0,0 +1,69 @@
+# Sending a Thank you
+def sendingthank(donors):
+    f_name = input('Please type the Full Name or list> ')
+    while f_name == 'list':
+        print(' ')
+        for x in donors:
+            print(x)
+        print(' ')
+        f_name = input('Please type the Full Name or list> ')
+    if donors.get(f_name, False):
```

Comment:
when using `.get` there is no need to provide the default value unless you're going to use that default value; in this case it isn't being used.

so you can just do `donors.get(f_name)` and `.get` will return None if it cannot find the key.